### PR TITLE
docs: Update README

### PR DESCRIPTION
**Pull request by BenitoVisone**

### CODE REVIEW

File: .github/workflows/code-review.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@ on:
     branches:
       - main
       - release-*
+    paths-ignore:
+      - "**.md"
+      - "**.yml"
```
Comment (rodoviti) [inline .github/workflows/code-review.yml:9]: why ignore only these two type of files and why all yml file in all subpaths?

i suggest to use an "allowed-list" strategy, so:
```suggestion
    paths:
      - "**/src/main/**"
      - "**pom.xml"
      - "Dockerfile"
      - ".github/workflows/code-review.yml"
```

File: .github/workflows/build-and-deploy.yml [hunk 2]
```diff
@@ -12,6 +12,9 @@ on:
     branches:
       - "main"
       - "release-*"
+    paths-ignore:
+      - "**.md"
+      - "**.yml"
```
Comment (rodoviti) [inline .github/workflows/build-and-deploy.yml:17]: why ignore only these two type of files and why all yml file in all subpaths?

i suggest to use an "allowed-list" strategy, so:
```suggestion
    paths:
      - "**/src/main/**"
      - "**pom.xml"
      - "Dockerfile"
      - ".github/workflows/build-and-deploy.yml"
```

**Review by rodoviti — CHANGES_REQUESTED** [review]

File: .github/workflows/build-and-deploy.yml (hunk 2 above)
Comment (BenitoVisone) [reply .github/workflows/build-and-deploy.yml]: fixed with https://github.com/pagopa/pdv-ms-person/pull/47/commits/2841ba68e2a42980577cf11d77944c9bc3dfae57

File: .github/workflows/code-review.yml (hunk 1 above)
Comment (BenitoVisone) [reply .github/workflows/code-review.yml]: fixed with https://github.com/pagopa/pdv-ms-person/pull/47/commits/2841ba68e2a42980577cf11d77944c9bc3dfae57

File: README.md [hunk 3]
```diff
@@ -1 +1,87 @@
-# pdv-ms-person 
+# PDV MS-Person
+
+Spring Application implementing the internal **Person** microservice for the **Personal Data Vault** Project.
+
+This microservice works in conjunction with:
+
+- [pdv-ms-tokenizer](https://github.com/pagopa/pdv-ms-tokenizer)
+- [pdv-ms-user-registry](https://github.com/pagopa/pdv-ms-user-registry)
```
Comment (rodoviti) [inline README.md:8]: from the point of view of the _tokenizer_, it is not aware of the existence of other microservices in the same _ecosystem_. So, I suggest splitting dependencies into upstream and downstream
```suggestion
---

## Dependencies

Upstream:
- [pdv-ms-user-registry](https://github.com/pagopa/pdv-ms-user-registry)

Downstream:
- [Amazon DynamoDB](https://aws.amazon.com/dynamodb/?nc1=h_ls)
```

Comment (BenitoVisone) [reply README.md:8]: in these lines I wanted to highlight that this micro-service is part of the Personal Data Vault project and to give a global point of view I added these two links to easily navigate them. I think that we could keep both the information.

```suggestion
This microservice is part of the Personal Data Vault Project and works in conjunction with:

- [pdv-ms-tokenizer](https://github.com/pagopa/pdv-ms-tokenizer)
- [pdv-ms-user-registry](https://github.com/pagopa/pdv-ms-user-registry)

## Dependencies
Upstream:
- [pdv-ms-user-registry](https://github.com/pagopa/pdv-ms-user-registry)
Downstream:
- [Amazon DynamoDB](https://aws.amazon.com/dynamodb/?nc1=h_ls)

```

File: README.md [hunk 4]
```diff
@@ -59,27 +61,23 @@ From terminal, inside the **app** package:
...
+| AWS_SECRET_ACCESS_KEY | AWS Secret Access Key *ppa-personal-data-vault-uat*[^aws_sso] |
+| AWS_SESSION_TOKEN     | AWS Session Token *ppa-personal-data-vault-uat*[^aws_sso]     |
 
-[^1]: For info about AWS SSO login, see [here](https://pagopa.atlassian.net/wiki/spaces/DEVOPS/pages/466846955/AWS+-+Users+groups+and+roles#Users-and-groups---DevOps-team).
+[^aws_sso]: For info about AWS SSO login, see [here](https://pagopa.atlassian.net/wiki/spaces/DEVOPS/pages/466846955/AWS+-+Users+groups+and+roles#Users-and-groups---DevOps-team).
 
 From terminal, inside the **app** package:
 
 ```
 ./mvnw spring-boot:run
```
Comment (BenitoVisone) [inline README.md:78]:
```suggestion
../mvnw spring-boot:run
```

**Review by rodoviti — APPROVED** [review]

**Pull request merged**